Failing action unit test without creators from NgRx schematic

When I use the action schematic to generate an action unit test with creators, I get a passing unit test. However, when I use the action schematic to generate an action unit test without creators, I get a failing unit test.

##Minimal reproduction of the bug/regression with instructions:

You get a failing spec when not using creators:
```
$ ng g @ngrx/schematics:action Add --project=counter --spec
$ npm test

ERROR in src/app/add.actions.spec.ts:1:10 - error TS2305: Module '"C:/Users/walrus95/proj/counter/src/app/add.actions"' has no exported member 'Add'.
1 import { Add } from './add.actions';
```

However, you get a passing spec when using creators:

```
$ ng g @ngrx/schematics:action Add --project=counter --spec --creators
$ npm test
TOTAL: 3 SUCCESS
```

## Expected behavior:

I would expect my newly created test to always be passing regardless of the use of creators, but at the very least it should be consistent.

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):

NgRx 8, Angular 8, Node 10.16.0

## I would be willing to submit a PR to fix this issue

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[x] No

Too busy swimming the arctic ocean. Splash, splash.

--- a/modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template
+++ b/modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template
@@ -1,7 +1,7 @@
-import { <%= classify(name) %> } from './<%= dasherize(name) %>.actions';
+import * as from<%= classify(name) %> from './<%= dasherize(name) %>.actions';
 
 describe('<%= classify(name) %>', () => {
   it('should create an instance', () => {
-    expect(new <%= classify(name) %>()).toBeTruthy();
+    expect(new from<%= classify(name)%>.Load<%= classify(name) %>s()).toBeTruthy();
   });
 });

--- a/modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template
+++ b/modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template
@@ -1,7 +1,7 @@
-import * as from<%= classify(name) %> from './<%= dasherize(name) %>.actions';
+import * as <%= classify(name) %>Actions from './<%= dasherize(name) %>.actions';
 
 describe('<%= classify(name) %>', () => {
   it('should create an instance', () => {
-    expect(new from<%= classify(name)%>.Load<%= classify(name) %>s()).toBeTruthy();
+    expect(new <%= classify(name)%>Actions.Load<%= classify(name) %>s()).toBeTruthy();
   });
 });

--- a/modules/schematics/src/action/index.spec.ts
+++ b/modules/schematics/src/action/index.spec.ts
@@ -122,7 +122,7 @@ describe('Action Schematic', () => {
         `${projectPath}/src/app/foo.actions.spec.ts`
       );
 
-      expect(fileContent).toMatch(/expect\(new fromFoo.LoadFoos\(\)\)/);
+      expect(fileContent).toMatch(/expect\(new FooActions.LoadFoos\(\)\)/);
     });
   });
 

--- a/modules/schematics/src/action/index.spec.ts
+++ b/modules/schematics/src/action/index.spec.ts
@@ -114,6 +114,16 @@ describe('Action Schematic', () => {
 
       expect(fileContent).toMatch(/export type FooActions = LoadFoos/);
     });
+
+    it('should create spec class with right imports', () => {
+      const options = { ...defaultOptions, spec: true };
+      const tree = schematicRunner.runSchematic('action', options, appTree);
+      const fileContent = tree.readContent(
+        `${projectPath}/src/app/foo.actions.spec.ts`
+      );
+
+      expect(fileContent).toMatch(/expect\(new fromFoo.LoadFoos\(\)\)/);
+    });
   });
 
   describe('action creators', () => {

